microservices-tetris-workflow-sizing/README.md

```diff
@@ -0,0 +1,7 @@
+# Fitting microservices into workflows like Tetris
+
+## Abstract
+
+In a microservice reality, components are small and focused but don’t provide much value in isolation. In these systems, workflows are where the magic happens: they string together multiple services to participate in a larger conversation that fulfills critical business processes. But how we view and scope these workflows matters. How do you define the scope of a single workflow? How does the workflow size relate to service boundaries? How does the scope affect decisions about coordination mechanisms? And how do you keep a larger overview when you decide to split things apart for the sake of loose coupling? 
```

Comment:
thought: maybe the abstract could mention something about the impact of poorly designed workflows onto the system and why the scoping is so important.

Reply:
This is a good idea, I need to think about how to incorporate this somehow.